packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -180,30 +178,33 @@ if (__DEV__) {
     didWarnStateUpdateForUnmountedComponent[componentName] = true;
   };
 
-  warnAboutInvalidUpdates = function(instance: React$Component<any>) {
+  warnAboutInvalidUpdates = function(fiber: Fiber) {
+    const componentName = getComponentName(fiber) || 'ReactClass';
```

Comment:
Let’s change this to “Unknown”? I know it’s not consistent but I like that better